packages/cashier/globals.d.ts

```diff
@@ -1,4 +0,0 @@
-declare module '@deriv/components';
```

Comment:
We have another card for cashier's TS configs issues (74387)

Reply:
Since no one start working on that I guess you can assign that to yourself and add this PR link to the card

Reply:
@niloo-fs Would be better to move them separately, We don’t know which one gonna get merged first, Probably `74387` will go first as it doesn’t need QA so we can have the correct config sooner. What do you say? 🤔

Reply:
Alright, agree with that.